setup.py

```diff
@@ -123,10 +112,10 @@
     classifiers=[
         'License :: OSI Approved :: MIT License',
         'Natural Language :: English',
-        'Programming Language :: Python :: 3.6',
-        'Programming Language :: Python :: 3.7',
-        'Programming Language :: Python :: 3.8',
```

Comment:
Not saying it's wrong, but is there any benefit of dropping also 3.8?

Reply:
Who will test it on python3.8 and why?

Reply:
Or we will try to make tmt code python3.8 compatible? I think that even python3.9 is quite old ;)

Reply:
I guess in Fedora/c9s/rhel9 world it's all 3.9+, but 3.8 is has more then one year until EOL. No opinion from my side, just raising questions :)
In rpm world, it probably doesn't make sense (unless someone wants to run it on rhel-7 with 3.8 from scl)
Though, tmt from pypi can be useful on other operating systems as far as I can tell... 

> Or we will try to make tmt code python3.8 compatible? 

Not sure there is anything to be done to be honest. I'm thinking more in a way of "3.8 can be dropped once it will require any effort / workarounds / changes compared to 3.9"

Reply:
> Not sure there is anything to be done to be honest. I'm thinking more in a way of "3.8 can be dropped once it will require any effort / workarounds / changes compared to 3.9"

Well, if we keep it it will immediately require effort - We will have to be sure that new language features are python3.8 compatible and if not we would need to fix the labels.

IMO safer and less work is to drop python3.8 support immediately. We don't want to be compatible.